### PR TITLE
Add UIView helper methods and documentation. Add default animation duration for cleaner syntax.

### PR DESCRIPTION
**Pull request by intrepidmatt**

### CODE REVIEW

File: Pod/Classes/AnimationSugar.swift [hunk 1]
```diff
@@ -104,3 +171,124 @@ public class Animation {
...
+    }
+    
+    /**
+     Fade in a view by animating its `alpha`.
+     
+     - Parameter duration: The duration of the animation
+     - Returns: An `Animation` that can be chained or altered.
+     */
+    func fadeIn(duration duration: NSTimeInterval = Animation.defaultAnimationDuration) -> Animation {
+        self.alpha = 0.0
```
Comment (loganwright) [inline Pod/Classes/AnimationSugar.swift:277]: Would it be worth calling `.hidden = false` here? Paul and I have been using a similar function that also turns off hidden. Might be a side-effect depending on how you think of it.

**Comment by loganwright** [comment]
:shipit:

File: Pod/Classes/AnimationSugar.swift (hunk 1 above)
Comment (intrepidmatt) [inline Pod/Classes/AnimationSugar.swift:277]: I think that assumes too much about how you've set up nibs/etc. Hidden or not is not part of an animation block, you'd set it either before or after the animation.

**Comment by ipbuildserver** [comment]
:ship:

**Comment by brightredchilli** [comment]
^ that was me

**Pull request merged**